cogs5e/sheets/dicecloudv2.py

```diff
@@ -29,10 +29,10 @@
 
 log = logging.getLogger(__name__)
 
-API_BASE = "https://beta.dicecloud.com/character/"
-DICECLOUDV2_URL_RE = re.compile(r"(?:https?://)?beta\.dicecloud\.com/character/([\d\w]+)/?")
+API_BASE = "https://dicecloud.com/character/"
+DICECLOUDV2_URL_RE = re.compile(r"(?:https?://)?((beta|app|www)\.)?dicecloud\.com/character/([\d\w]+)/?")
```

Comment:
Does this work locally? Its currently adding two groups that match, so could mess things up

```suggestion
DICECLOUDV2_URL_RE = re.compile(r"(?:https?://)?(?:(?:beta|app|www)\.)?dicecloud\.com/character/([\d\w]+)/?")
```

Reply:
Oh I forgot about that